Library/Formula/gtk+.rb

```diff
@@ -12,7 +12,9 @@ class Gtkx < Formula
     sha256 "3f3948ce65bd2c199e2603a8bebc39f6de14e3c7f408226b1d6a0c8b52a51f29" => :mountain_lion
   end
 
-  option "with-quartz-relocation", "Build with quartz relocation support"
+  if OS.mac?
+    option "with-quartz-relocation", "Build with quartz relocation support"
+  end
```

Comment:
Please remove this chunk. I prefer a minimall patch, to make merging with Homebrew easier.